stage2/00-sys-tweaks/00-packages

```diff
@@ -1,10 +1,10 @@
 ssh less fbset sudo psmisc strace ed ncdu crda
 console-setup keyboard-configuration debconf-utils parted unzip
-manpages-dev python bash-completion pkg-config
```

Comment:
How sure are you that python2 is not required anywhere? Just from your familiarity of Raspberry Pi OS or are you using something to check?

Reply:
I used a bash script to find all packages that should be installed, then used apt-rdepends to check all dependencies, and checked if python2 is listed.

I don't think anything should still depend on it if it isn't in Debian dependencies, because this would violate Debian guidelines

Reply:
Ok, great!